### DISCUSSION TITLE
Caddy plugin of interest

### DISCUSSION THREAD

**Discussion by cweagans** [pockethost/pockethost#264] [Q&A]
Hi there! I thought you might be interested in knowing about https://github.com/cweagans/caddy_ondemand_upstreams. It's a caddy plugin that I wrote for the purpose of hosting multiple pockebase instances (and other applications) easily. It seems to do something similar to your proxy component and also handles all of the other stuff that Caddy handles (automatic certs and all that). Might be of interest to you :)

**Comment by benallfree** [comment]
Thanks for sharing this! What is Caddy?

Does your plug-in shut down on idle?

**Comment by benallfree** [comment]
https://github.com/caddyserver/caddy

**Comment by cweagans** [comment]
Caddy is a web server. The plugin shuts stuff down on idle. I still need to figure out how to handle websocket connections appropriately, but it works as expected for normal http requests right now.

**Comment by benallfree** [comment]
PocketBase uses SSE but no websockets. I suppose a frontend might, though? 

Please consider sending a PR to awesome-PocketBase as well!

**Comment by cweagans** [comment]
Ah okay. Well either way, it's tracking activity on the initial connection and not ongoing data transfer over the same connection. Haven't figured that out yet.
